docs/reference/generated/checkbox-root.json

```diff
@@ -26,6 +26,10 @@
       "default": "false",
       "description": "Whether the checkbox is in a mixed state: neither ticked, nor unticked."
     },
+    "value": {
+      "type": "number | string",
+      "description": "The value of the selected checkbox."
+    },
```

Comment:
`CheckboxGroup` uses `name` to link, but maybe it should now be `value` (which is like `RadioGroup`)

Reply:
From my experience it's not very common to set values on checkboxes (at least when posting to a server) - it's the name what's important and the value is just the default `on`/`off`. With radio buttons, there is no other way than using values.
I'd personally leave it as is.

One thing that we can consider, though, is the customization of the unchecked value. Currently, it's hardcoded to `off`. I never needed to customize it myself, but maybe there is a use case. We can wait for community feedback on it.